filters/tee/tee.go

```diff
@@ -17,9 +19,16 @@ const (
 	NoFollowName   = "teenf"
 )
 
+const defaultTeeTimeout = 1 * time.Second
+
 type teeSpec struct {
 	deprecated bool
-	noFollow   bool
+	options    Options
+}
+
+type Options struct {
+	noFollow bool
```

Comment:
this way the fields of the option structure are not exported. Should be `NoFollow` and `Timeout`.